airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/retriers/retrier.py

```diff
@@ -20,9 +36,5 @@ def retry_factor(self) -> float:
         pass
 
     @abstractmethod
-    def should_retry(self, response: requests.Response) -> bool:
-        pass
-
-    @abstractmethod
-    def backoff_time(self, response: requests.Response) -> Optional[float]:
```

Comment:
this method isn't needed anymore as should_retry returns a ResponseStatus that might contain the backoff time